src/http.rs

```diff
@@ -838,9 +881,15 @@ fn api_status(origins: &OriginsHistory) -> Response<Body> {
                         None => target.member_raw("session", "none")
                     }
                     target.member_raw("delta",
-                        if metrics.delta { "true" }
+                        if metrics.snapshot_reason.is_none() { "true" }
                         else { "false" }
                     );
+                    if let Some(reason) = metrics.snapshot_reason {
+                        target.member_str("snapshot_reason", reason.code())
+                    }
+                    else {
+                        target.member_raw("snapshot_reason", "none");
```

Comment:
I don't think `none` is a valid json value

Reply:
Argl. Fixed.